.eslintrc.yml

```diff
@@ -1 +1,31 @@
-extends: 'eslint-config-node-services'
\ No newline at end of file
+extends: 
+  - 'eslint-config-wikimedia/server'
+
+plugins:
+  - json
+  - jsdoc
+
+rules:
+  array-bracket-spacing:
+    - off
+  camelcase:
+    - error
+    - properties: never
+  computed-property-spacing:
+    - off
+  indent: 
+    - off
```

Comment:
Actually, why is it off? We do want to validate indent, just want to override tabs to spaces.

Reply:
https://github.com/wikimedia/change-propagation/pull/302#pullrequestreview-194255471

Reply:
This is different from [the one in `rb-m-t-spec`](https://github.com/wikimedia/restbase-mod-table-spec/pull/53/files#diff-0378d5111ab996d5882f4712e9350604R13). While both have the same effect, we should probably use the same values to minimise confusion.